app/assets/javascripts/discourse/app/initializers/jquery-plugins.js

```diff
@@ -8,6 +10,36 @@ export default {
     bootbox.animate(false);
     bootbox.backdrop(true);
 
+    // Monkey-patching simple alerts
+    const originalAlert = bootbox.alert;
+    bootbox.alert = function () {
+      if (arguments.length === 1) {
+        const dialog = getOwner(this).lookup("service:dialog");
+        if (dialog) {
+          deprecated(
+            "`bootbox.alert` is deprecated, please use the dialog service instead.",
+            {
+              dropFrom: "3.0.0.beta1",
+            }
+          );
+          return dialog.alert(arguments[0]);
+        }
+      }
+      return originalAlert(...arguments);
+    };
+
+    // adding deprecation notice for all other dialogs
+    const originalDialog = bootbox.dialog;
+    bootbox.dialog = function () {
+      deprecated(
+        "`bootbox` is now deprecated, please use the dialog service instead.",
+        {
+          dropFrom: "3.0.0.beta1",
```

Comment:
That's probably wrong 😅 There either won't be 3.0.0.beta1 or it will be too soon for the drop to happen (i.e. it will be this beta cycle)

Reply:
Yes, good point. Probably should be `3.0.0.beta5`, i.e. we'll drop it halfway between stable releases 3.0 and 3.1?

Reply:
That would be `3.0.1.beta5` (though beta1 is fine too)

Reply:
Hmm... should it in fact be `3.1.0.beta5`? Since we're skipping `2.9`, the next stable release is going to be `3.0` and the first beta after that _should_ be `3.1.0.beta1`, right? So, if we were to target halfway through the cycle between 3.0 and 3.1 stables, that would mean `3.1.0.beta5`. 

Also, more generally, it doesn't look like we ever update the tiny version number.